rtdb-shipengine-validate-address/functions/.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
Is it not possible to put these config files at the repo root? Do they need to be in each individual extension folder?

Reply:
Ah actually no, it looks like I can drop the extra eslintrc files

Reply:
Eslint dropped from extension folders. Lerna/yarn update forthcoming in its own PR.